### DISCUSSION TITLE
Cant get my discount

### DISCUSSION THREAD

**Discussion by Parz1val2045** [community/community#56264] [GitHub Education]
### Hi everyone,

Hi everyone

I hope you can help me here, i cant get past the request a discount. It says i have the following problems:

![image](https://github.com/community/community/assets/68967836/0dac6554-9bf7-4159-8a3f-a5cb13b77665)

But if you look at my profile, i have a short bio my full name and a profile picture. Can someone please help me with whats wrong?

**Comment by samus-aran** [comment]
Hi @Parz1val2045 :wave:

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
